recipes/xtl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xtl" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "97cb01feb042ee59727ea1dcf502ed300c3533f009a6e606094b2811c66ee1f7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xtl/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/xtl  # [unix]
+    - test -f ${PREFIX}/include/xtl/xbasic_fixed_string.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xtl/xtlConfig.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xtl/xtlConfigVersion.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\xtl\xbasic_fixed_string.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\xtlConfig.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\xtlConfigVersion.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: http://quantstack.net
```

Comment:
This should probably be https://github.com/QuantStack/xtl, since quantstack.net seems to be the generic company page.

Reply:
Most of the other packages in the QuantStack org have a quantstack.net/ProjectName page, such as `xtensor`, `xsimd`, `xproperty`, `xeus`...

xtl is a library of utilities that are used by all of these, and are progressively separated into their own package. It will soon have a page there.